pkgs/build-support/agda/default.nix

```diff
@@ -90,8 +90,6 @@ let
       pname,
       meta,
       buildInputs ? [ ],
-      everythingFile ? "./Everything.agda",
-      includePaths ? [ ],
```

Comment:
Is `includePaths` also not necessary anymore? I guess this information is in the `.agda-lib` file?

Reply:
Indeed, `agda --build-library` just ignores any `-i` arguments.